src/components/CommentList.jsx

```diff
@@ -0,0 +1,27 @@
+import { useState, useEffect } from "react";
+import { useParams } from "react-router-dom";
+import CommentCard from "./CommentCard";
+import { fetchComments } from "../api-utils"
+
+const CommentList = () => {
+  const { article_id } = useParams();
+
+  const [commentData, setCommentData] = useState([]);
+
+  useEffect(() => {
+    fetchComments(article_id).then((res) => setCommentData(res));
+  }, [article_id]);
+
+  return (
+    <div>
+      <h1>Comments</h1>
+      <ul>
+        {commentData.map((comment) => {
```

Comment:
In both your maps currently you're getting an error in the console about the `key` prop - you'll need to pass a key to each comment and each article